src/dialog/facerecogniser.py

```
@@ -0,0 +1,59 @@
+from __future__ import annotations
+
+import time
+
+import cv2
+import face_recognition
+from numpy._typing import NDArray
+from memory.memorymanager import MemoryManager
+from typing import Tuple
+
+
+class FaceRecogniser:
+    def __init__(self, memory: MemoryManager):
+
+        self.memory = memory
+
+        # Create an object to read camera video
+        self.cap = cv2.VideoCapture(0)
+
+        # Check if camera opened successfully
+        if not self.cap.isOpened():
+            print("Camera is unable to open.")
```

I think it might be good to raise an error instead of just printing. Then we'll be sure to know if something went wrong

---

done